docs/developer-guide.md

```diff
@@ -0,0 +1,280 @@
+
+  // Must correspond to a containerVendor record with the same org_id value.
+  "org_id": <insert-org-id>,
+
+  "container": {
+    "type": "operator bundle image",
+
+    // Required but always "rhcc"
+    "distribution_method": "rhcc",
+
+    // Always set to true to satisfy the publishing checklist
+    "distribution_approval": true,
+
+    // Must match the github user(s) which opened the test pull requests
+    "github_usernames": ["<insert-github-username>"],
+
+    // Must be unique for the vendor
+    "repository_name": "<insert-repo-name>"
+  }
+}
+```
+
+#### Prepare an Operator Bundle
+
+You can use a utility script to copy an existing bundle. By default it will copy
+a bundle that should avoid common failure conditions such as digest pinning.
+View all the customization options by passing `-h` to this command.
+
+```bash
+./scripts/copy-bundle.sh
+```
+
+You may wish to tweak the generated output to influence the behavior of the pipelines.
+For example, Red Hat Marketplace Operator bundles may require additional annotations.
+The pipeline should provide sufficient error messages to indicate what is missing.
+If such errors are unclear, that is likely a bug which should be fixed.
+
+#### Prepare Your ci.yaml
+
+At the root of your operator package directory (note: **not** the bundle version directory)
+there needs to be a `ci.yaml` file. For development purposes, it should follow
+this format in most cases.
+
+```yaml
+---
+# Copy this value from the _id field of the certification project in Pyxis.
+cert_project_id: <pyxis-cert-project-id>
+# Set this to true to allow the hosted pipeline to merge pull requests.
+merge: false
+```
+
+#### Create a Bundle Pull Request
+
+It's recommended to open bundle pull requests against the
+[operator-pipelines-test](https://github.com/redhat-openshift-ecosystem/operator-pipelines-test)
+repo. The pipeline GitHub bot account has permissions to manage it.
+
+> **Note:** This repository is only configured for testing certified operators,
+> NOT Red Hat Marketplace operators (see
+> [`config.yaml`](https://github.com/redhat-openshift-ecosystem/operator-pipelines-test/blob/main/config.yaml)).
+
+```bash
+# Checkout the pipelines test repo
+git clone https://github.com/redhat-openshift-ecosystem/operator-pipelines-test
+cd operator-pipelines-test
+
+# Create a new branch
+git checkout -b <insert-branch-name>
+
+# Copy your package directory
+cp -R <package-dir>/ operators/
+
+# Commit changes
+git add -A
+
+# Use this commit pattern so it defaults to the pull request title.
+# This is critical to the success of the pipelines.
+git commit -m "operator <operator-package-name> (<bundle-version>)"
+
+# Push your branch. Open the pull request using the output.
+git push origin <insert-branch-name>
+```
+
+> **Note:** You may need to merge the pull request to use it for testing the release pipeline.
+
+#### Create an API Key
+
+Contact a Pyxis admin to assist with this request. It must correspond to the
```

Comment:
Maybe add a link to file an ISV ticket here?

Reply:
Added a link to the internal catalog page.